bindings/python/examples/how_tos/account_wallet/request_funds.py

```diff
@@ -3,23 +3,26 @@
 
 from dotenv import load_dotenv
 
-from iota_sdk import Wallet, Utils, SyncOptions, AccountSyncOptions
+from iota_sdk import Wallet, WalletOptions, Utils, SyncOptions, WalletSyncOptions
 
-# In this example we request funds to an account wallet.
+# In this example we request funds to a wallet.
```

Comment:
This was correct, since it's about an account output
Also the changes in the prints need to be reverted